File: Pesquisa e Ordenação/aula_01_31.07/ordenacao_python/Principal.py
from Util import Util
lista_numeros = []
lista2_numeros = []

lista_palavras = []


Util.popular_aleatorio_numeros(lista_numeros, 10, 100, 500)
Util.exibir_lista(lista_numeros)

lista2_numeros.extend(lista_numeros)

Util.popular_aleatorio_palavras(lista_palavras, 10, 6)
Util.exibir_lista(lista_palavras)

File: Pesquisa e Ordenação/aula_01_31.07/ordenacao_java/Principal.java

package ordenacao;

import java.util.ArrayList;



public class Principal {
    public static void main(String[] args) {
        ArrayList<Integer> listaNumeros = new ArrayList<>();
        ArrayList<String> listaPalavras = new ArrayList<>();
        
        Util.popularAleatorioNumeros(listaNumeros, 10,100,500);
        Util.exibirListaNumeros(listaNumeros);
        
        Util.popularAleatorioPalavras(listaPalavras, 10, 6);
        Util.exibirListaPalavras(listaPalavras);

        
        
    }
}
